Makefile

```diff
@@ -201,11 +201,13 @@ godocs:
 
 build-docs:
 	@cd docs && \
-	while read p; do \
-		(git checkout $${p} && npm install && VUEPRESS_BASE="/$${p}/" npm run build) ; \
-		mkdir -p ~/output/$${p} ; \
-		cp -r .vuepress/dist/* ~/output/$${p}/ ; \
-		cp ~/output/$${p}/index.html ~/output ; \
+	while read -a p; do \
+		branch=$${p[0]} ; \
+		path_prefix=$${p[1]} ; \
+		(git checkout $${branch} && npm install && VUEPRESS_BASE="/$${path_prefix}/" npm run build) ; \
+		mkdir -p ~/output/$${path_prefix} ; \
+		cp -r .vuepress/dist/* ~/output/$${path_prefix}/ ; \
+		cp ~/output/$${path_prefix}/index.html ~/output ; \
```

Comment:
Since there are 2 lines in `versions` now, the 2nd index.html (launchpad's) will override the 1st one (master's).

I think we should put the master's index.html as the root one.